### ISSUE TITLE
Any way to use this app instead when triggering the macOS Color Picker?

### ISSUE THREAD

**Issue by jonathanlaniado** [sindresorhus/System-Color-Picker#6]
I love this app, thank you! But in Preview, if I choose the color picker, it opens up the macOS system color picker, not this one. Is there any way to accomplish that?

**Comment by sindresorhus** [comment]
No. However, I plan to make a color picker plugin with the text fields that are in this app, but it's not going to happen in the short term. I want to make this app great first.

For now, you could try one of the existing color picker plugins mentioned in the readme.

Another workaround would be to pick the color in this app, drag it into the palette at the bottom, which is then available in the system color picker too.
